spanish language locale

CarlosVMX1 opened the pull request:
## Description
Added es.json file to the locale folder to use spanish language.

## Motivation and Context
There was not spanish file so I added it.

## How Has This Been Tested?
I had been using it for some days, all works correct.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

CarlosVMX1 committed:
spanish language locale

PokeMapGo commented:
Just a quick note:

-Keep the initial capital letter i.e: _"All team's": "todos los equipos"_
-Use the original game translation i.e: 
_"Make 5 Great Curveball Throws in a row": "Haz 5 lanzamientos Geniales en curva seguidos"_ ---> _Haz 5 grandes lanzamientos de bola curva seguidos_
-This is not the same: _"404 No valid invasion types found": "Error de comando",_

Anyway, great job, I always forget to PR this locale

CarlosVMX1 commented:
Hey @PokeMapGo if you have a better spanish locale I’m happy to remove this one, added it only because almost all my friends plays on spanish and I was not able to find that translation so I shared mine, but I see that the text translated from the game are not accurate with the original ones so need to change all of them, and for example I translate the 404 ones like that because I did it based on the de.json file and is a literal translation, didn’t know the original one don’t have the 404 thing.

CarlosVMX1 committed:
Update es.json v2

-Removed all text already translated at the gamemaster file src/util/locale/es.json
-Checked all translations to meet the rules.

jfberry commented:
Since I don't speak spanish I have no way of verifying. Just let me know when ready to merge

jfberry commented:
@CarlosVMX1 what is the status of this?

jfberry commented:
Spanish language support added to PoracleNG
